android/src/test/java/com/revenuecat/purchases/hybridcommon/PurchaserInfoMappersTests.kt

```diff
@@ -1,16 +1,14 @@
-package com.revenuecat.purchases.common
+package com.revenuecat.purchases.hybridcommon
 
 import android.net.Uri
 import com.revenuecat.purchases.PurchaserInfo
-import com.revenuecat.purchases.common.mappers.map
-import com.revenuecat.purchases.common.mappers.toIso8601
-import com.revenuecat.purchases.common.mappers.toMillis
+import com.revenuecat.purchases.hybridcommon.mappers.toIso8601
+import com.revenuecat.purchases.hybridcommon.mappers.toMillis
+import com.revenuecat.purchases.hybridcommon.mappers.*
```

Comment:
Is this needed?

Reply:
this might have been just me trying stuff out until something worked and forgetting to revert. I'll try reverting